src/components/FeaturePanel/FeaturePanel.tsx

```diff
@@ -28,10 +28,30 @@ const Flex = styled.div`
   flex: 1;
 `;
 
-export const FeaturePanel = () => {
+type FeaturePanelProps = {
+  onHeadingHeightChange?: (height: number) => void;
+};
+
+export const FeaturePanel = ({ onHeadingHeightChange }: FeaturePanelProps) => {
   const { feature } = useFeatureContext();
   const [advanced, setAdvanced] = useState(false);
   const [showTags, toggleShowTags] = useToggleState(false);
+  const headingRef = useRef<HTMLDivElement>();
+
+  useEffect(() => {
+    const headingDiv = headingRef.current;
+    if (!headingDiv) return;
+
+    const listener = () => {
+      onHeadingHeightChange?.(headingDiv.clientHeight);
+    };
+    listener();
+
+    window.addEventListener('resize', listener);
+    return () => {
+      window.removeEventListener('resize', listener);
+    };
+  }, [headingRef, feature, onHeadingHeightChange]);
```

Comment:
1. could we move this useEffect only to the `FeaturePanelInDrawer.tsx` and pass headingRef only? It would not be used for FeaturePanel-on-side.
2. ~~to simplyfi the code - I would be OK with computing this only once (not on every `resize`). Screen width of a phone hardly changes.~~ oh wait, it changes in portrait/landscape mode. It didn't occur to me first. Let's at least add a comment to the listener, that it is here to accomodate orientation. It will help every other reader who thinks as I do. Or even better, call the fn `handlePortraitLandscapeChange` instead of `listener` - that is the selfdocumenting code i wrote about above. 🙂

Reply:
My thought was more like a small window on a computer which gets resized, most modern phones are big enough to use the drawer only in portrait mode.

Reply:
> most modern phones are big enough to use the drawer only in portrait mode.

You are right. So only issue is resizing the small window on desktop. This is an edge case TBH. Either way is OK – leave listener or remove it as you wish. My thinking is "less code, less bugs", so I would probably remove it. But both solutions are good.

Comment could go like `only for resizing a small window on a desktop, landscape on mobile is usually wide enough to show standard FeaturePanel`